.github/workflows/backend-ci.yml

```diff
@@ -77,6 +77,11 @@ jobs:
       - uses: docker-practice/actions-setup-docker@master
         timeout-minutes: 12
       - run: ./scripts/start_local_db.sh
+      - name: setup binstall
+        uses: taiki-e/install-action@cargo-binstall
+      - name: Install forc for tests
+        run: |
+          cargo binstall --no-confirm --root tests/fixtures/forc-0.65.0 forc@0.65.0
```

Comment:
Should the version string be dynamic so it doesn't have to be touched every update?

Reply:
There's an argument for it but usually we don't want tests to start failing because of a release in another repo.